app/calendar/event-creator.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+import { useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+export default function EventCreator({ username, onEventCreated }) {
```

Comment:
I moved the file in another package. The only difference is that this component has a new parameter (username) which is used in handleClick() method